lib/vanagon/component/dsl.rb

```diff
@@ -50,6 +50,10 @@ def method_missing(method, *args)
         @component.send(attribute)
       end
 
+      def respond_to_missing?(method_name, include_private = false)
```

Comment:
No, because it's additive/complimentary to a rewrite of the corresponding `method_missing`